### DISCUSSION TITLE
Integer Primary key for Users

### DISCUSSION THREAD

**Discussion by artemdzyuba** [supabase/supabase#3327] [Questions] [answered]
HiI I Use Wappler no-code builder, and for joining tables involving users it assumes that the Primary key is integer while it is `UUID`.

If it can't be changed or updated then I have to simple create my own users table with the integer primary key. And will have to give up all the Supabase auth functionality.

Is it possible to switch from `UUID` to the `Integer` type?

**Comment by silentworks** [comment, marked as answer]
No, you can't change the `UUID` to and `INT` on the supabase `auth.users` table, but you can create a table called `profiles` in the `public` schema and add an `INT` as a primary key on it. You would still need to have the `UUID` field for the relationship to the `auth.users` table though.

**Reply by artemdzyuba** [reply]
Thnak you. You mean, I can still use `auth.users` for authentification, and 1-to-1 relationship with `profiles` for operations with a `INT` primary key?

**Reply by silentworks** [reply]
Yes, that's correct.

**Reply by artemdzyuba** [reply]
Thank you
